### ISSUE TITLE
folding failed during the optimize process.

### ISSUE THREAD

**Issue by jinz2014** [intel/intel-extension-for-pytorch#500]
### Describe the issue

Hi,
I ran the example from https://intel.github.io/intel-extension-for-pytorch/xpu/latest/tutorials/examples.html 
Do the warning message indicate some issue ?  Thanks.

/home/users/user/anaconda3/envs/lib/python3.10/site-packages/intel_extension_for_pytorch/frontend.py:611: UserWarning: Conv BatchNorm folding failed during the optimize process.
  warnings.warn(
/home/users/user/anaconda3/envs/lib/python3.10/site-packages/intel_extension_for_pytorch/frontend.py:618: UserWarning: Linear BatchNorm folding failed during the optimize process.
  warnings.warn(
Execution finished

**Comment by jingxu10** [comment]
Hi, this warning is not an issue. it indicates this folding is not applicable. It doesn't affect other optimizations.

**Comment by jinz2014** [comment]
Okay.
